enterprise/backend/src/metabase_enterprise/advanced_config/api/logs.clj

```diff
@@ -37,7 +37,7 @@
   Must be a superuser."
   [yyyy-mm]
   {yyyy-mm (mu/with-api-error-message [:re #"\d{4}-\d{2}"]
-             (deferred-tru "Must be a string like 2020-04 or 2222-11."))}
+                                      (deferred-tru "Must be a string like 2020-04 or 2222-11."))}
```

Comment:
This seems like a good case for a line break before the first arg. Maybe it's just me, but having a deep indent for a "with-" form, even if its a function, looks kinda funky.